docs/fundamentals/executor/instrumenting-executor.md

```diff
@@ -0,0 +1,272 @@
+(instrumenting-executor)=
+# Instrumentation
+
+Instrumentation consists of [OpenTelemetry](https://opentelemetry.io) Tracing and Metrics. Each feature can be enabled independently, and they allow you to collect request-level and application level metrics for analyzing an Executor's real-time behavior. 
+
+This section documents creating custom traces and metrics.
```

Comment:
I'm not sure we need lines like these. We literally discussed traces and metrics in the last para, so users are primed already